### ISSUE TITLE
eslint errors not jumping to locations

### ISSUE THREAD

**Issue by mqudsi** [neomake/neomake#1581]
Using the default eslint maker (`Neomake eslint`) with Neomake 8794ee654a715867d34045af49dd00e8474b0850 under neovim v0.2.1-855-gcb912a3 results in the linter correctly catching errors and their line numbers being displayed in the the location list, but attempting to jump to an error does nothing.

What shows up in the location list:

```
cpsatv.js|1 col 1 warning| Definition for rule 'react/jsx-sort-prop-types' was not found (react/jsx-sort-prop-types)
cpsatv.js|1 col 1 warning| Definition for rule 'react/require-extension' was not found (react/require-extension)
cpsatv.js|1 col 1 warning| Definition for rule 'react/wrap-multilines' was not found (react/wrap-multilines)
cpsatv.js|38 col 58 error| Expected '===' and instead saw '=='. (eqeqeq)
cpsatv.js|64 col 12 warning| Gratuitous parentheses around expression. (no-extra-parens)
cpsatv.js|64 col 19 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|64 col 40 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|102 col 17 warning| Gratuitous parentheses around expression. (no-extra-parens)
cpsatv.js|102 col 35 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|102 col 53 warning| Gratuitous parentheses around expression. (no-extra-parens)
cpsatv.js|105 col 17 warning| Gratuitous parentheses around expression. (no-extra-parens)
cpsatv.js|105 col 36 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|105 col 54 warning| Gratuitous parentheses around expression. (no-extra-parens)
cpsatv.js|108 col 25 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|144 col 34 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|169 col 98 warning| Missing semicolon. (semi)
cpsatv.js|177 col 19 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|215 col 22 error| Expected '===' and instead saw '=='. (eqeqeq)
cpsatv.js|218 col 12 error| Expected '!==' and instead saw '!='. (eqeqeq)
cpsatv.js|222 col 2 error| Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
cpsatv.js|223 col 2 error| Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
cpsatv.js|227 col 27 error| Expected '===' and instead saw '=='. (eqeqeq)
cpsatv.js|230 col 27 error| Expected '===' and instead saw '=='. (eqeqeq)
cpsatv.js|232 col 40 warning| Missing semicolon. (semi)
cpsatv.js|236 col 27 error| Expected '===' and instead saw '=='. (eqeqeq)
cpsatv.js|237 col 7 error| 'div' is already defined. (no-redeclare)
cpsatv.js|240 col 27 error| Expected '===' and instead saw '=='. (eqeqeq)
cpsatv.js|244 col 4 warning| Missing semicolon. (semi)
cpsatv.js|249 col 24 error| Expected '===' and instead saw '=='. (eqeqeq)
cpsatv.js|251 col 7 error| 'lines' is already defined. (no-redeclare)
cpsatv.js|255 col 7 error| 'lines' is already defined. (no-redeclare)
```

The first three fake results (they aren't an error with the actual file, but general warnings from eslint) jump to location 1:1 fine, but from the 3rd error onwards, nothing happens.

The NeomakeInfo output is below:

```
#### Neomake information (use ":verbose NeomakeInfo" for extra output)                                                                                   ##### Enabled makers                                                                                                                                                                         For the current filetype ("javascript", used with :Neomake):                                                                                                                                  - jshint                                                                                                                                                                                       - append_file: 1                                                                                                                                                                             - args: ['--verbose']                                                                                                                                                                        - auto_enabled: 1                                                                                                                                                                            - errorformat: '%A%f: line %l\, col %v\, %m \(%t%*\d\)'                                                                                                                                      - exe: 'jshint'                                                                                                                                                                              - postprocess: function('neomake#postprocess#GenericLengthPostprocess')                                                                                                                      - WARNINGS:                                                                                                                                                                                    - maker's exe (jshint) is not executable.                                                                                                                                                - jscs                                                                                                                                                                                         - append_file: 1                                                                                                                                                                             - args: ['--no-colors', '--reporter', 'inline']                                                                                                                                              - auto_enabled: 1                                                                                                                                                                            - errorformat: '%E%f: line %l\, col %c\, %m'                                                                                                                                                 - exe: 'jscs'                                                                                                                                                                              - eslint                                                                                                                                                                                       - append_file: 1                                                                                                                                                                             - args: ['-f', 'compact']                                                                                                                                                                    - auto_enabled: 1                                                                                                                                                                            - errorformat: '%E%f: line %l\, col %c\, Error - %m,%W%f: line %l\, col %c\, Warning - %m,%-G,%-G%*\d problems%#'                                                                            - exe: 'eslint'                                                                                                                                                                           NOTE: you can define g:neomake_javascript_enabled_makers to configure it (or b:neomake_javascript_enabled_makers).                                                                                                                                                                                                                                                                        For the project (used with :Neomake!):                                                                                                                                                        - makeprg                                                                                                                                                                                      - args: []                                                                                                                                                                                   - auto_enabled: 0                                                                                                                                                                            - errorformat: '%*[^"]"%f"%*\D%l: %m,"%f"%*\D%l: %m,%-G%f:%l: (Each undeclared identifier is reported only once,%-G%f:%l: for each function it appears in.),%-GIn file included from %f:%l:%c:,%-GIn file included from %f:%l:%c\,,%-GIn file included from %f:%l:%c,%-GIn file included from %f:%l,%-G%*[ ]from %f:%l:%c,%-G%*[ ]from %f:%l:,%-G%*[ ]from %f:%l\,,%-G%*[ ]from %f:%l,%f:%l:%c:%m,%f(%l):%m,%f:%l:%m,"%f"\, line %l%*\D%c%*[^ ] %m,%D%*\a[%*\d]: Entering directory %*[`'']%f'',%X%*\a[%*\d]: Leaving directory %*[`'']%f'',%D%*\a: Entering directory %*[`'']%f'',%X%*\a: Leaving directory %*[`'']%f'',%DMaking %*\a in %f,%f|%l| %m'                                                                                                                             - exe: 'makeprg'                                                                                                                                                                             - fn: function('7')                                                                                                                                                                          - ERRORS:                                                                                                                                                                                      - maker's exe (makeprg) is not executable.                                                                                                                                              NOTE: you can define g:neomake_enabled_makers to configure it.                                                                                                                                                                                                                                                                                                                            Default maker settings:                                                                                                                                                                       - remove_invalid_entries: 0                                                                                                                                                                  - buffer_output: 1                                                                                                                                                                                                                                                                                                                                                                       ##### Settings                                                                                                                                                                               ```                                                                                                                                                                                          g:neomake_cpp_clangtidy_maker = {'errorformat': '%E%f:%l:%c: fatal error: %m,%E%f:%l:%c: error: %m,%W%f:%l:%c: warning: %m,%-G%\m%\%%(LLVM ERROR:%\|No compilation database found%\)%\@!%.%#,%E%m', 'args': ['--', '-fsyntax-only', '-Wall', '-Wextra', '-I./', '-std=c++1z', '-I./'], 'exe': 'clang-tidy'}                                                                               g:neomake_cpp_clangxx_maker = {'errorformat': '%-G%f:%s:,%f:%l:%c: %trror: %m,%f:%l:%c: %tarning: %m,%I%f:%l:%c: note: %m,%f:%l:%c: %m,%f:%l: %trror: %m,%f:%l: %tarning: %m,%I%f:%l: note: %m,%f:%l: %m', 'args': ['-fsyntax-only', '-Wall', '-Wextra', '-I./', '-std=c++1z', '-I./'], 'exe': 'clang++'}                                                                                 g:neomake_cpp_enabled_makers = ['clangxx', 'clangtidy']                                                                                                                                      g:neomake_enabled_makers = ['makeprg']                                                                                                                                                       g:neomake_makeprg_maker = {'errorformat': '%*[^"]"%f"%*\D%l: %m,"%f"%*\D%l: %m,%-G%f:%l: (Each undeclared identifier is reported only once,%-G%f:%l: for each function it appears in.),%-GIn file included from %f:%l:%c:,%-GIn file included from %f:%l:%c\,,%-GIn file included from %f:%l:%c,%-GIn file included from %f:%l,%-G%*[ ]from %f:%l:%c,%-G%*[ ]from %f:%l:,%-G%*[ ]from %f:%l\,,%-G%*[ ]from %f:%l,%f:%l:%c:%m,%f(%l):%m,%f:%l:%m,"%f"\, line %l%*\D%c%*[^ ] %m,%D%*\a[%*\d]: Entering directory %*[`'']%f'',%X%*\a[%*\d]: Leaving directory %*[`'']%f'',%D%*\a: Entering directory %*[`'']%f'',%X%*\a: Leaving directory %*[`'']%f'',%DMaking %*\a in %f,%f|%l| %m', 'args': ['-j4'], 'exe': 'make'}                                                                 g:neomake_open_list = 1                                                                                                                                                                      g:neomake_place_signs = 1                                                                                                                                                                                                                                                                                                                                                                   makeprg=make                                                                                                                                                                               ```

**Comment by blueyed** [comment]
Duplicate of #1582?

**Comment by mqudsi** [comment]
Oops. GitHub timed out so I refreshed and it seemed to have not gone through.
